src/System.Net.Sockets/tests/FunctionalTests/Accept.cs

```diff
@@ -97,6 +97,13 @@ public abstract class Accept<T> : SocketTestHelperBase<T> where T : SocketHelper
         [InlineData(5)]
         public async Task Accept_ConcurrentAcceptsAfterConnects_Success(int numberAccepts)
         {
+            // The SyncForceNonBlocking implementation currently toggles the listener's Blocking setting
+            // back and force on every Accept, which causes pending sync Accepts to return EWOULDBLOCK.
+            // For now, just skip the test for SyncForceNonBlocking.
+            // TODO: Refactor SyncForceNonBlocking Accept handling to avoid this problem
```

Comment:
Is there an issue number associated with this TODO that could be included in the comment?